src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerAgreementControllerTests.cs

```diff
@@ -68,79 +68,7 @@ public Task ConfirmRemoveOrganisation_WhenIRemoveAlegalEntityFromAnAccount_ThenT
                     Assert.IsNotNull(actualModel);
                     Assert.IsNotNull(actualModel.FlashMessage);
                 });
-        }
```

Comment:
I do not know how to make this diff go away, it is nonsense. The file is checked out at parent branch state. Local Git diff shows no diffs.